components/team_general_tab/team_general_tab.tsx

```diff
@@ -2,56 +2,61 @@
 // See LICENSE.txt for license information.
 /* eslint-disable react/no-string-refs */
 
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ChangeEvent, MouseEvent, ReactNode} from 'react';
 import {FormattedMessage, FormattedDate} from 'react-intl';
 
 import Constants from 'utils/constants';
-import * as Utils from 'utils/utils';
+import {imageURLForTeam, isMobile, localizeMessage, moveCursorToEnd} from 'utils/utils';
+import {t} from 'utils/i18n';
+
 import SettingItemMax from 'components/setting_item_max.jsx';
 import SettingItemMin from 'components/setting_item_min';
 import SettingPicture from 'components/setting_picture.jsx';
 import BackIcon from 'components/widgets/icons/fa_back_icon';
 import LocalizedInput from 'components/localized_input/localized_input';
-import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
-import {t} from 'utils/i18n';
+import {Team} from '@mattermost/types/teams';
 
-import OpenInvite from './open_invite';
+import OpenInvite from './open_invite/open_invite';
```

Comment:
I think we can avoid this by putting the files back where they were originally. It would make sense to do this if the component was tied to Redux and needed a connector, but since it's standalone I think we can leave it alone.

Reply:
Thanks, I will keep that in mind for future PRs. 

It is just a personal preference that if there are more than 1 file (i.e., `open_invite.tsx` and `open_invite.test.tsx` in this case) related to same component, I would like to organize them in a separate folder. It is not a big issue, I reverted the changes as suggested.